setup.py

```diff
@@ -15,7 +15,8 @@
 REQUIRED_PKGS = [
     "coloredlogs",
     "sympy",
-    "transformers[sentencepiece]>=4.29",
+    "transformers @ git+https://github.com/huggingface/transformers.git",
```

Comment:
```suggestion
    "transformers[sentencepiece] @ git+https://github.com/huggingface/transformers.git",
```
cc @IlyasMoutawwakil